pkg/intermediate/aggregate.go

```diff
@@ -237,15 +237,15 @@ func (a *AggregationProcess) correlateRecords(incomingRecord, existingRecord ent
 				if ieWithValue.Value != "" {
 					existingIeWithValue, _ := existingRecord.GetInfoElementWithValue(field)
 					if existingIeWithValue.Value != "" {
-						klog.Warningf("This field with name %v should not have been filled with value %v in existing record.", field, existingIeWithValue.Value)
+						klog.Warningf("This field with name %v and value %v should not have been filled with value %v in existing record.", field, ieWithValue.Value, existingIeWithValue.Value)
```

Comment:
This log is not clear. If we need current value, lets do this:
" %v field should not have been filled in the existing record; existing value: %v and current value: %v"

Reply:
Done. Thanks!